src/server/ApiRoutes/Search.js

```diff
@@ -129,6 +129,7 @@ function fetchResults(searchKeywords = '', contributor, title, subject, page, so
         itemListElement.forEach((resultObj) => {
           const { result } = resultObj;
           const items = result.items;
+          if (!items) return results;
```

Comment:
i'm not sure if this is addressing the problem of bibs with no items. bibs with no items still have an items array, it's just empty. this might solve the bug immediately, but warrants further investigation.